<launch>

  <arg name="UAV_NAME" default="$(env UAV_NAME)"/>

  <group ns="$(arg UAV_NAME)">
    
    <node name="TrajectoryFlier" pkg="nodelet" type="nodelet" args="standalone mrs_testing/TrajectoryFlier" output="screen" >
      
      <rosparam file="$(find mrs_testing)/config/trajectory_flier.yaml" />

      <!-- Subscribers -->
      <remap from="~control_cmd_in" to="control_manager/cmd_odom" />

      <!-- Services -->
      <remap from="~activate_in" to="~activate" />
      <remap from="~trajectory_out" to="control_manager/mpc_tracker/set_trajectory" />

    </node>
  
  </group>
</launch>
